solr/core/src/java/org/apache/solr/handler/admin/ClusterStatus.java

```diff
@@ -147,6 +191,7 @@ public void getClusterStatus(NamedList<Object> results)
 
     NamedList<Object> collectionProps = new SimpleOrderedMap<>();
 
+    // loop will run only once
```

Comment:
Why is that?

Reply:
Was based on an earlier revision, removed it now